code/modules/mob/living/carbon/brain/posibrain.dm

```diff
@@ -23,7 +23,7 @@ var/global/posibrain_notif_cooldown = 0
 
 /obj/item/device/mmi/posibrain/proc/ping_ghosts(msg)
 	if(!posibrain_notif_cooldown)
-		notify_ghosts("Positronic Brain [msg] in [get_area(src)]. <a href=?src=\ref[src];activate=1>(Enter)</a>")
+		notify_ghosts("<span class='ghostalert'>Positronic brain [msg] in [get_area(src)]!</span> <a href=?src=\ref[src];activate=1>(Click to enter)</a>", 'sound/effects/ghost2.ogg')
```

Comment:
The span classes are not needed, as the notify_ghosts() proc already sets the 'ghostalert' span.